cmd/runc-wrapper/runc-wrapper.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2017 Kinvolk GmbH
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"log"
+	"os"
+	"os/exec"
+)
+
+const (
+	runc        string = "/usr/libexec/docker/docker-runc-current"
+	logLocation string = "/tmp/nspawn-runc-wrapper.log"
```

Comment:
This seems to be for debugging. Could we do that only if the user requests debugs?

Is it possible to have an environment variable that optionally sets up the log file? I don't know if the user can enable/disable an environment variable in `/etc/docker/daemon.json`...